minecraft_servers/domei/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "server_name": "domei",
+  "nice_name": "DOMEI",
+  "direct_ip": "domei.eu",
+  "server_wildcards": [
+    "%.domei.eu"
+  ],
+  "supported_languages": [
+     "fr"
+  ],
+  "social": {
+    "web": "https://domei.eu/",
+    "web_shop": "https://domei.eu/boutique/",
+    "twitter": "DomeiMc",
+    "discord": "https://discord.com/invite/domei",
+    "youtube": "https://www.youtube.com/channel/UCOV9bf8jJ973hjg17zXMhAQ"
+  },
+  "gamemodes": {
+    "uhc": {
+      "name": "UHC",
+      "color": "#FFFF55"
+    },
+    "lg-uhc": {
+      "name": "Loup-Garou UHC",
+      "color": "#FF5555"
+    },
+    "jjk-uhc": {
+      "name": "Jujutsu Kaisen UHC",
+      "color": "#AA00AA"
+    },
+    "ds-uhc": {
+      "name": "DemonSlayer UHC",
+      "color": "#FFAA00"
+    },
+    "cg-uhc": {
+      "name": "Code Geass UHC",
+      "color": "#FFFFFF"
+    },
+    "naruto-uhc": {
+      "name": "Naruto UHC",
+      "color": "#FFAA00"
+    },
+    "nakimeparty": {
+      "name": "NakimeParty",
+      "color": "#FF5555"
+    },
+    "abyssdown": {
+      "name": "AbyssDown",
+      "color": "#55FF55"
+    },
+    "ssb": {
+      "name": "Super Smash Bows",
+      "color": "#FFFF55"
+    }
+  },
+  "discord": {
+    "server_id": 837316390708445204,
+    "rename_to_minecraft_name": true
```

Comment:
Please use "false" as it is the default value. "true" is currently only available for partners.

Reply:
Haven't see this, thanks!